apps/migrations/0015_tagulous_abstractcustomappinstance_tags_and_more.py

```diff
@@ -0,0 +1,129 @@
+                    models.BooleanField(default=False, help_text="Will not be deleted when the count reaches 0"),
+                ),
+            ],
+            options={
+                "ordering": ("name",),
+                "abstract": False,
+                "unique_together": {("slug",)},
+            },
+            bases=(tagulous.models.models.BaseTagModel, models.Model),
+        ),
+        migrations.CreateModel(
+            name="Tagulous_GradioInstance_tags",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("name", models.CharField(max_length=255, unique=True)),
+                ("slug", models.SlugField()),
+                (
+                    "count",
+                    models.IntegerField(default=0, help_text="Internal counter of how many times this tag is in use"),
+                ),
+                (
+                    "protected",
+                    models.BooleanField(default=False, help_text="Will not be deleted when the count reaches 0"),
+                ),
+            ],
+            options={
+                "ordering": ("name",),
+                "abstract": False,
+                "unique_together": {("slug",)},
+            },
+            bases=(tagulous.models.models.BaseTagModel, models.Model),
+        ),
+        migrations.CreateModel(
+            name="GradioInstance",
+            fields=[
+                (
+                    "baseappinstance_ptr",
+                    models.OneToOneField(
+                        auto_created=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        parent_link=True,
+                        primary_key=True,
+                        serialize=False,
+                        to="apps.baseappinstance",
+                    ),
+                ),
+                (
+                    "logs_enabled",
+                    models.BooleanField(
+                        default=True, help_text="Indicates whether logs are activated and visible to the user"
+                    ),
+                ),
+                ("note_on_linkonly_privacy", models.TextField(blank=True, default="", null=True)),
+                ("source_code_url", models.URLField(blank=True, null=True)),
+                ("description", models.TextField(blank=True, default="", null=True)),
+                (
+                    "access",
+                    models.CharField(
+                        choices=[
+                            ("project", "Project"),
+                            ("private", "Private"),
+                            ("public", "Public"),
+                            ("link", "Link"),
+                        ],
+                        default="project",
+                        max_length=20,
+                    ),
+                ),
+                ("port", models.IntegerField(default=8000)),
```

Comment:
Should not the default port for Gradio be 8080 and also displayed as 8080 in the Gradio create form?